### ISSUE TITLE
update replaces script contents with "404 Not Found"

### ISSUE THREAD

**Issue by J-Reis** [pystardust/ani-cli#798]
**Metadata (please complete the following information)**
Version: was 3.2
OS: Linux (Debian Stable)
Shell: fish
Anime: N/A

**Describe the bug**
After update with `sudo ani-cli -U` ani-cli no longer executes. In fact the actual content of the script is now just 404: Not Found. v3 was working before update.

**Steps To Reproduce**

1. update using `sudo ani-cli -U` → update messages appear as normal
2. attempt to start ani-cli with `ani-cli`
3. error message received: `exec: Exec format error` in fish or `/usr/local/bin/ani-cli: line 1: 404:: command not found` in bash

**Expected behavior**
`ani-cli` shows search prompt

**Screenshots (if applicable; you can just drag the image onto github)**
![2022-06-25_18-33-52](https://user-images.githubusercontent.com/23666144/175793904-1a5af426-635f-42a0-a56e-03d2ef4deddb.png)

**Additional context**

**Comment by realitymolder** [comment]
same issue also on mac

**Comment by Derisis13** [comment]
That definitely wasn't v3.2, I can tell from the location of the files that this was an instance of v3.1. The reason the updating failed is because we changed the structure (again) in v3.2 to better to fit into the linux/unix/whatever filesystem.
To fix this reinstall the script (follow instructions in the readme or look for packages compatible with your os).

**Comment by realitymolder** [comment]
As @Derisis13, it helped. Uninstalling with brew only didn't help, you should go to the README on the main project and delete the script with your platform specific method. 
after doing so, reinstall, and it should go away.

**Comment by realitymolder** [comment]
OK wait, after this comment I tried watching something, didn't work. 
got an error : 
`Loading episode 22...
Invalid character in input stream.
bad decrypt
4311828012:error:06FFF06D:digital envelope routines:CRYPTO_internal:wrong final block length:/AppleInternal/Library/BuildRoots/b6051351-c030-11ec-96e9-3e7866fcf3a1/Library/Caches/com.apple.xbs/Sources/libressl/libressl-2.8/crypto/evp/evp_enc.c:512:
Video URL not found`

So I did ani-cli -U. 
script updated. 

Testing again... 
`ani-cli one piece
/opt/homebrew/bin/ani-cli: line 1: 404:: command not found`

**Comment by J-Reis** [comment]
> That definitely wasn't v3.2, I can tell from the location of the files that this was an instance of v3.1 … 

Right, my mistake. At some point during the various breakdowns since v3 I remember opening the script in vim and seeing `VERSION="3.2.something"` in the header comments. But, I do have multiple machines and a termux set-up; each of which I only got around to fixing as I wanted to use them; so that must have been under slightly different circumstances. Of course, the version on this machine was impossible to check once everything had been overwritten with 404's.

As I mentioned, removal & re-installation did indeed work for me, so everything is ship shape on my end. Only posted the issue in case it helped to flag something. Unfortunately, I haven't got any insight to share regarding **realitymolder's** issues.
